src/doc/unstable-book/src/language-features/rustc-attrs.md

```diff
@@ -0,0 +1,50 @@
+# `rustc_attrs`
+
+This feature has no tracking issue, and is therefore likely internal to
+the compiler, not being intended for general use.
+
+------------------------
+
+The `rustc_attrs` feature allows debugging rustc type layouts by using
```

Comment:
This is true but not precise. `rustc_attrs` enables *tons* of attributes, namely all of the ones starting with `rustc_`.

Reply:
I don't know about the others, I only know about `rustc_attrs`. I cannot be precise here without the knowledge, I already wrote what I knew. I also never wrote that it `rustc_layout` is the only macro provided by `rustc_attrs`.

Reply:
You could add a preamble saying that the feature enables many rustc-internal attributes, and that this page only discusses a few of them.

Reply:
The current text kind of makes it sound like the attribute is specific to `rustc_layout`; I think we text can be improved to avoid making that suggestion. I can suggest concrete wordings if you want.

Reply:
I have added a note.